central/node/datastore/store/postgres/cve_cache.go

```diff
@@ -0,0 +1,59 @@
+package postgres
+
+import (
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/sync"
+)
+
+// nodeCVECache is a thread-safe in-memory cache for NodeCVE objects
+type nodeCVECache struct {
+	mu    sync.RWMutex
+	cache map[string]*storage.NodeCVE
+}
+
+// newNodeCVECache creates a new thread-safe NodeCVE cache
+func newNodeCVECache() *nodeCVECache {
+	return &nodeCVECache{
+		cache: make(map[string]*storage.NodeCVE, 256),
+	}
+}
+
+func (c *nodeCVECache) GetMany(ids []string) (map[string]*storage.NodeCVE, []string) {
```

Comment:
General comment: How about we use sliceutils.* functions to reduce the boilerplate where possible?